README.md

```diff
@@ -276,3 +276,5 @@ make manifests
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
 </details>
+
+
```

Comment:
Can you update the README to match your additions ? Among which : 

- new User API/CRD
- how it works
- ties with K8S secrets 
- ...